templates/jsonfeed/shell.tpl.php

```diff
@@ -72,48 +72,24 @@
 
             $feedItem['content_html'] = $item->getBody();
 
-            $feedItem['_indieweb'] = array();
+            $feedItem['_meta'] = array();
             if (method_exists($item, 'getMetadataForFeed')) {
-                $feedItem['_indieweb'] = $item->getMetadataForFeed();
-            } else if ($item instanceof \IdnoPlugins\Like\Like) {
+                $feedItem['_meta'] = $item->getMetadataForFeed();
```

Comment:
Good question, @mapkyca. I looked into using `jsonSerialize()` for this purpose, but for the same reasons that I moved this serialization from `templates/json/shell.tpl.php` to `templates/jsonfeed/shell.tpl.php`, I think its best to keep these two things separate. While both methods are generating fields for serialization in JSON, they are doing so for very different purposes (API consumption vs. JSON Feed generation). The current uses of `jsonSerialize()` and the API may have different requirements from consumers of JSON Feed.

I can add a sensible default to `Entity` easily, since I already have the basis for it inside of  `templates/jsonfeed/shell.tpl.php`. I'll just move it. Stand by!